src/redis_metadata.cc

```diff
@@ -232,11 +232,14 @@ RedisType Metadata::Type() const {
 
 int32_t Metadata::TTL() const {
   int64_t now;
+  if (expire <= 0) {
+    return -1;
+  }
```

Comment:
What's the meaning of `-1` and `-2` returns here? As @mapleFU described here change the result `expire <0 && expire < now` from `-2` to `-1`.

Reply:
https://github.com/apache/incubator-kvrocks/blob/unstable/src/redis_db.cc#L146
one comment explained `-2` here, far away from this function though :rofl: